### PR TITLE
[dualtor_io]: Fix BGP tests

### PR DESCRIPTION
**Pull request by theasianpianist**
Signed-off-by: Lawrence Lee <lawlee@microsoft.com>

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
Test expectations did not align with new image behavior

#### How did you do it?
- Change the expected active and standby ToRs for some scenarios
- Remove tunnel traffic check for some scenarios

#### How did you verify/test it?
Run BGP tests, verify they pass 
```
=== Running tests in groups ===
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now
deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
=================================================================================== test session starts ====================================================================================platform linux2 -- Python 2.7.17, pytest-4.6.5, py-1.10.0, pluggy-0.13.1
ansible: 2.8.12
rootdir: /var/src/sonic-mgmt/tests, inifile: pytest.ini
plugins: forked-1.3.0, metadata-1.11.0, xdist-1.28.0, html-1.22.1, repeat-0.9.1, ansible-2.2.2
collected 4 items

dualtor_io/test_tor_bgp_failure.py::test_active_tor_kill_bgpd_upstream PASSED                                                                                                        [ 25%]
dualtor_io/test_tor_bgp_failure.py::test_standby_tor_kill_bgpd_upstream PASSED                                                                                                       [ 50%]
dualtor_io/test_tor_bgp_failure.py::test_standby_tor_kill_bgpd_downstream_active PASSED                                                                                              [ 75%]
dualtor_io/test_tor_bgp_failure.py::test_active_tor_kill_bgpd_downstream_standby PASSED                                                                                              [100%]

---------------------------------------------------------------- generated xml file: /var/src/sonic-mgmt/tests/logs/tr.xml -----------------------------------------------------------------=============================================================================== 4 passed in 2254.30 seconds ================================================================================
```

### CODE REVIEW

File: tests/dualtor_io/test_tor_bgp_failure.py [hunk 1]
```diff
@@ -97,16 +97,14 @@ def test_active_tor_kill_bgpd_downstream_standby(
     Action: Shutdown all BGP sessions on the active ToR
     Expectation:
         Verify packet flow after the active ToR (A) loses BGP sessions
-        T1 switch continues to receive IP-in-IP traffic, from lower to upper ToR
-        No switchover occurs
-        verify traffic interruption is < 1 second
+        Verify ToR A standby, ToR B active
+        Verify traffic interruption is < 1 second
     '''
-    with tunnel_traffic_monitor(lower_tor_host, existing=True):
```
Comment (vaibhavhd) [inline tests/dualtor_io/test_tor_bgp_failure.py:104]: Why is tunnel traffic verification removed?

**Comment by vaibhavhd** [comment]
@theasianpianist, I have merged the PR, but please double check if the tunnel verification check is really needed to be removed. Thanks.

File: tests/dualtor_io/test_tor_bgp_failure.py (hunk 1 above)
Comment (theasianpianist) [reply tests/dualtor_io/test_tor_bgp_failure.py:104]: Newer 202012 images include a mechanism to initiate switchover if BGP goes down. Thus, at the beginning of the test the standby should be forwarding downstream packets to the active via IPinIP. However, once the test kills the bgpd process on the active, a switchover should be triggered which means the old standby is now the new active, and should not be sending any IPinIP traffic, i.e. we would see tunnel traffic for the first part of the test only. I don't believe the tunnel monitor has a way to account for this state change in the middle of the test, which is why I removed it.

**Pull request merged**